module/metrics/execution.go

```diff
@@ -497,15 +505,15 @@ func (ec *ExecutionCollector) ExecutionBlockExecuted(dur time.Duration, compUsed
 // ExecutionCollectionExecuted reports computation and total time spent on a block computation
 func (ec *ExecutionCollector) ExecutionCollectionExecuted(dur time.Duration, compUsed uint64, txCounts int) {
 	ec.totalExecutedCollectionsCounter.Inc()
-	ec.collectionExecutionTime.Observe(float64(dur))
+	ec.collectionExecutionTime.Observe(float64(dur / time.Millisecond))
```

Comment:
we should be able to just do `dur.Milliseconds()` which might be a bit more clear

```suggestion
	ec.collectionExecutionTime.Observe(float64(dur.Milliseconds()))
```